sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -165,38 +148,6 @@
     'WriteToBigQuery',
     ]
 
-JSON_COMPLIANCE_ERROR = 'NAN, INF and -INF values are not JSON compliant.'
-MAX_RETRIES = 3
-
-
-def default_encoder(obj):
-  if isinstance(obj, decimal.Decimal):
-    return str(obj)
-  raise TypeError(
-      "Object of type '%s' is not JSON serializable" % type(obj).__name__)
-
-
-class RowAsDictJsonCoder(coders.Coder):
```

Comment:
Unfortunately I think this (and some of the other changes below) are backwards incompatible changes to the public API. Can we leave stubs to those methods with deprecation warnings so that we don't break backwards compatibility ?